### PR TITLE
Change: Use xcf2png instead of GIMP

### PR DESCRIPTION
**Pull request by Milek7**
Contrary to previous gimpscript which preserved layer order from .xcf file, xcf2png composes layers in specified order. Only situation when this yields different output is in waterfeatures.xcf2png, so this is fixed.
With Xcftools 1.0.7 it gives pixel-identical results as .png commited in repository.
Curiously GIMP 2.10 gives different output because of some blending changes, but I guess it won't matter now.

### CODE REVIEW

**Comment by TrueBrain** [comment]
Confirmed that the md5sum and sha256sum of the produced GRFs are identical with and without this patch (given `NEWGRF_VERSION` is set to the same value). LGTM.

File: README.md [hunk 1]
```diff
@@ -122,9 +122,6 @@ Your system should already have most tools, you'll probably only need NML and gi
 Install the developers tools and get NML from the source mentioned above. git is easiest installed via MacPorts:
 `sudo port install git`
 
-On OSX GIMP is not found in the path, if you installed the app package as supplied from the GIMP's project page. You can add that to your search path if you link the binary which requires the X-environment to be running:
-`sudo ln /Applications/Gimp.app/Contents/Resources/bin/gimp /usr/local/bin/gimp`
```
Comment (TrueBrain) [inline README.md]: Pretty sure OSX people want their hand hold; ask an OSX user where xcftools can be found? (brew? port?)

**Comment by glx22** [comment]
It's a good idea, but the main issue is xcftools are not really available outside linux it seems

**Comment by kaechele** [comment]
I'm somewhat opposed to this change as xcftools is unmaintained by the original maintainer and has known bugs.
There is a fork of xcftools available at https://github.com/j-jorge/xcftools where you can also check the issue tracker for the known bugs (including some arbitrary code execution exploits).
Furthermore xcftools was removed as a package from Fedora about a year ago because it is unmaintained and has unfixed security vulnerabilities.
As the Fedora maintainer of both openttd and openttd-opengfx I would at least have to drop opengfx from the repos or fix-up xcftools to be acceptable for re-introduction into Fedora repos (as you probably figured that's not going to happen).
So my ask would be to make xcftools optional for those who want to use it and still have gimp support for those who have to use it.

**Comment by TrueBrain** [comment]
Not disagreeing with what you wrote, just an observation:

For the record, gimp (and xcftools with this PR) are always optional. I don't see any real need why you, as package maintainer, would want either :) The PNGs it produces are stored in the repository, and should, in my opinion, be used by package maintainers. As that means everyone gets the same package. Otherwise it would, for example by different GIMP versions, that Fedora has another OpenGFX than, say, Debian.

Of course, for -dev package I can understand the need. But for OpenGFX I doubt anyone would use it :)

**Comment by planetmaker** [comment]
as sprite generation from xcf to png is optional and a maintainer thing, keeping gimp seems more reasonable: a maintainer for this graphics set will be in need of gimp. Using gimp as cmd-line tool works on a CI/CD just fine, too, if you use it as additional integration check. If anyone wants to use xcf2png, one can do so by setting GIMP=xcf2png and GIMP_FLAGS= - and it could be even done on a CI/CD.

**Comment by LordAro** [comment]
Given the difficulty in obtaining xcf2png on some systems, and it's possible unmaintained status, I'm going to close this for now.

**Pull request closed without merging**